src/dissect/dissect.c

```diff
@@ -898,21 +923,28 @@ static int action_dissect(DissectedImage *m, LoopDevice *d) {
                 printf("            %s initrd\n",
                        COLOR_MARK_BOOL(!strv_isempty(m->initrd_release)));
 
-                r = get_extension_scopes(m, &extension_scopes);
+                r = (image_class == IMAGE_SYSEXT) ? get_sysext_scopes(m, &sysext_scopes) : get_confext_scopes(m, &confext_scopes);
                 if (r < 0)
                         return log_error_errno(r, "Failed to parse scope: %m");
 
-                printf("            %s extension for system\n",
-                       COLOR_MARK_BOOL(strv_contains(extension_scopes, "system")));
-                printf("            %s extension for initrd\n",
-                       COLOR_MARK_BOOL(strv_contains(extension_scopes, "initrd")));
-                printf("            %s extension for portable service\n",
-                       COLOR_MARK_BOOL(strv_contains(extension_scopes, "portable")));
+                printf("            %s sysext extension for system\n",
```

Comment:
it cannot be both at the same time, so merge the new lines into the existing ones